server.js

```diff
@@ -1,28 +1,31 @@
 const cors = require('cors');
 const express = require('express');
 const passport = require('passport');
+const mongoose = require('mongoose');
 
-const app = express();
+const { APP, DB } = require('./config');
 
-const { APP } = require('./config');
-const auth = require('./middleware/auth');
 const authRouter = require('./routes/auth');
-const usersRouter = require('./routes/users');
-const incomesRouter = require('./routes/incomes');
-const expensesRouter = require('./routes/expenses');
 const accountsRouter = require('./routes/accounts');
+const transactionsRouter = require('./routes/transactions');
+
+const auth = require('./middleware/auth');
 const permissionHandler = require('./middleware/permission-handler');
 
+const app = express();
+const mongoUrl = `${DB.MONGODB_URL}${DB.DATABASE_NAME}`;
```

Comment:
Instead of concatenation of 2 variables you can store the whole MONGO_CONNECTION_STR in ENV